packages/netlify-cms-widget-list/src/ListControl.js

```diff
@@ -168,6 +175,18 @@ export default class ListControl extends React.Component {
     onChange((value || List()).push(parsedValue));
   };
 
+  processObjectControlRef = ref => {
```

Comment:
I'd stick with just `processControlRef`, I don't see the need to "namespace" the function

```suggestion
  processControlRef = ref => {
```